golang.md

```diff
@@ -0,0 +1,119 @@
+# Golang at Clever
+
+## Golang versions
+
+All golang services (with some exceptions) at Clever should be on go-1.5.
+
+
+## Docker Images
+
+#### drone
+
+`clever/drone-go:1.5`
+
+#### runtime
+
+  - `gliderlabs/alpine:3.2` - see [catapult/Makefile](https://github.com/Clever/catapult/blob/master/Makefile) as an example. (extra build configuration needed)
```

Comment:
Add some information about how to make one choice vs. another. If that is not trivial, then I would propose just saying that `debian:wheezy` should be used.